Jenkinsfile

```diff
@@ -342,6 +355,40 @@ pipeline {
             }
 
         }
+        stage('Math functions expressions test') {
+            when {
+                expression {
+                    !skipExpressionTests
+                }
+            }
+            agent any
+            steps {
+
+                sh """
+                    git clone --recursive https://github.com/stan-dev/stan.git
```

Comment:
You only need the Math library.

Reply:
So only clone math to somewhere and then move the bin/stanc and that should be it.